config.json

```diff
@@ -47,6 +47,26 @@
       "stepCount": 5
     },
 
+    "astracliGettingStarted": {
```

Comment:
I would capitalize "cli" in the guide name to be in line with the method used for other guides. astraCLIGettingStarted.